bsvNew.py

```diff
@@ -15,6 +15,12 @@ def is_dir_empty(path):
     else:
         return False
 
+def is_legal_name(project_name):
+    if project_name.find('-') != -1 or project_name == 'Action' or project_name == 'ActionValue' or project_name == 'valueOf':
```

Comment:
Thank you for the PR!

This check could be a bit more generic, e.g.:
```python
keywords = ["Action", "ActionValue"]
if '-' in project_name or project_name in keywords or project_name[0].islower():
```

Reply:
Maybe there is a list somewhere with other prohibited keywords.

Reply:
Thanks for the fast reply. 

I didn't think in this generic form! I agree it would looks more readable.